docs/src/guides/organizing-stores.mdx

```diff
@@ -134,12 +134,10 @@ Using the [provider][1] package on pub, you can setup a repository of stores at
 In the example below, we are setting up a store at the app-level:
 
 ```
-final MultiCounterStore store = MultiCounterStore();
-
 class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) => Provider<MultiCounterStore>(
-      value: store,
+      builder: (_) => MultiCounterStore(),
```

Comment:
Also I am thinking of going for a `MultiProvider`, as each example may have something to "provide" to others deeper in the example widget tree

Reply:
Was about to do that but I realized: where is coming from the `examples` variable used on `routes` argument of `MaterialApp`?